### PR TITLE
Enhance getting started guide

### PR DESCRIPTION
**Pull request by ARolek**

### CODE REVIEW

File: layouts/partials/head-meta.html [hunk 1]
```diff
@@ -16,6 +16,6 @@
 <meta name="yandex-verification" content="{{ . }}" />
 {{ end }}
 
-{{- .Hugo.Generator -}}
```
Comment (gdey) [inline layouts/partials/head-meta.html:19]: Are you sure about this change? 

Seems you are going from a context value to a function? But shouldn't a function be :
```
{{- hugo .Generator -}}
```

Don't know hugo, just from what I know of html/template.

Comment (ARolek) [reply layouts/partials/head-meta.html:19]: Yeah it was deprecated in v0.55. This is the suggested update approach: https://discourse.gohugo.io/t/pages-hugo-is-deprecated-as-of-0-55-0/17991

**Comment by mapl** [comment]
LGTM! - tested with the latest Hugo Version 0.80

**Review by mapl — APPROVED** [review]
LGTM! - tested with the latest Hugo Version 0.80

**Pull request merged**